app/lib/dttp/odata.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Dttp
+  class Odata
```

Comment:
Still on the fence about this class. I wanted to use the code in more that once place and decided to extract it for that reason. Testing the code behaviour became easier but feedback welcome on another approach or even just renaming the class - `Odata` does seem generic.

Reply:
I'd opt to renaming something more specific. This class' job seems to be parsing the odata response so `OdataParser` or something alone those lines?